tests/internal/e2e/test/examples/CCAST_.CFG

```diff
@@ -0,0 +1,106 @@
+ASSEMBLER_CMD: 
```

Comment:
Do not commit this; it needs to come via running `$VECTORCAST_DIR/clicast -lc template GNU_CPP_X`.

Reply:
Switched in aa9998331e53e037d2bd5cc34baf890e31934863 to use $VECTORCAST_DIR_TEST_DUPLICATE as prefix.